client/src/containers/App/index.js

```diff
@@ -18,12 +18,17 @@ class AppContainer extends Component {
     <App
       todoCards={this.state.todoCards}
       addTodoCard={this.addTodoCard}
+      deleteTodoCard={this.deleteTodoCard}
     />
   )
 
   addTodoCard = (card) => {
     this.setState(state => ({ todoCards: [...state.todoCards, card] }))
   }
+
+  deleteTodoCard = (cardId) => {
+    this.setState(state => ({ todoCards: state.todoCards.filter(el => el.id !== cardId) }))
```

Comment:
@AleksSenkou i think it's not so long. Any way which one do you prefer:
1.```
 this.setState(state =>
      ({ todoCards: state.todoCards.filter(el => el.id !== cardId) }))

```
2.```
    this.setState(
      state => ({ todoCards: state.todoCards.filter(el => el.id !== cardId) }),
    )
```

Reply:
@DDKatch hm, what do you think about this way:
```js
deleteTodoCard = cardId =>
  this.setState({todoCards} => { todoCards: this._filteredCards(todoCards, cardId) })

_filteredCards = (cards, cardId) =>
  cards.filter(card => card.id !== cardId)
```

method with prefix `_` used for showing that it's private // @datarockets/cuties

Reply:
@AleksSenkou I don't like it, even this line even longer than mine :wink: . Even in naming I prefer my variant, because `_filteredCards` is a really extensive name.

Reply:
@DDKatch we can go without line splitting then 👍 

So we'll split that when we need to change logic of filtering cards